testsuite/cluster-test/src/suite.rs

```diff
@@ -57,6 +58,7 @@ impl ExperimentSuite {
         experiments.push(Box::new(
             CpuFlamegraphParams { duration_secs: 60 }.build(cluster),
         ));
+        experiments.push(Box::new(RebootClusterParams {}.build(cluster)));
```

Comment:
We have some flackiness in nightly right now - do you mind removing it from suite until we figure out those issues, and adding it here a bit later?